_includes/links.md

```diff
@@ -4,8 +4,11 @@
 [0002]: {{site.baseurl}}/recipe/0002-mvm-audio/ "Simplest Manifest - Audio"
 [0003]: {{site.baseurl}}/recipe/0003-mvm-video/ "Simplest Manifest - Video"
 [0007]: {{site.baseurl}}/recipe/0007-string-formats/ "Embedding HTML"
+[0009]: {{site.baseurl}}/recipe/0009-book-1/ "Multiple Related Images (Book, etc.)"
```

Comment:
Could you update the title to "Simple Manifest - Book"

Reply:
Done in #209